src/multibody/model.hpp

```diff
@@ -100,9 +100,13 @@ namespace se3
     std::vector<Frame> frames;
     
     /// \brief Vector of subtrees.
-    /// subtree[j] corresponds to the subtree supported by the joint j.
+    /// subtrees[j] corresponds to the subtree supported by the joint j.
```

Comment:
I think this is not a marginal feature. It will become necessary in the future for very sparse algo which will deal with only certain links, as the computation of (JM-1Jt). And it is information cheap, easy to compete when building.

It can also be used to make smart update, etc.